app/views/responses/form.html.erb

```diff
@@ -6,13 +6,19 @@
   ELMO.fileUploaderManager = new ELMO.Views.FileUploaderManager({el: ".response-form"});
 <% end %>
 
+<% enketo_link = link_to(
+  t(%(new edit).include?(params[:action]) ? "response.edit_with_enketo" : "response.view_with_enketo"),
+  params.permit!.merge("enketo": "1")) %>
```

Comment:
i know this is server side rendered, but permit! makes me a lil nervous. not sure though if actually a risk here.

Reply:
this would certainly be dangerous if being passed to something like `user.update(params.permit!)`, but this is just being used to create an HTML link so it should be safe.